cmd/deployIntegrationArtifact_generated_test.go

```diff
@@ -0,0 +1,16 @@
+package cmd
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestDeployIntegrationArtifactCommand(t *testing.T) {
+	t.Parallel()
+	testCmd := DeployIntegrationArtifactCommand()
```

Comment:
Add empty line here to make the tests pass again. Or generate the files again...